handler/population_types.go

```diff
@@ -46,11 +54,13 @@ func (h *PopulationTypes) Get(w http.ResponseWriter, req *http.Request) {
 	log.Info(ctx, "population types found", logData)
 	lpt := len(ptypes)
 
+	paginated, limit, offset := manualPagination(r.Limit, r.Offset, ptypes)
```

Comment:
You're not getting the limit returned here you're getting limit+offset. I don't think you need to return these values at all